ronan_solutions/most_common_element_impl.py

```diff
@@ -8,6 +9,28 @@ def most_common_element(elements: List[Any]) -> List[Any]:
     there are three 0s, three 3s, two 2s, one 1 and one 4, so the correct solution is [0, 3]
     or [3, 0].
     """
-    return [None]
+    #I created a dictionary because I wanted to store not only the number of times a value appears, 
+    # but also the value itself at the same time
+    countPerItem = {}
+    #iteration over the list
+    for i in List:
```

Comment:
`List` here is actually just a type hint for the parameter `elements` (see line 5 above -- you might have to expand the "diff" view). You can't iterate over a type. So this should be `for i in elements`